src/server/kiwix-serve.cpp

```diff
@@ -423,14 +425,14 @@ static struct MHD_Response* handle_suggest(RequestContext* request_context)
   mimeType = "application/json; charset=utf-8";
   bool deflated = request_context->acceptEncodingDeflate && compress_content(content, mimeType);
   return build_response(
-      content.data(), content.size(), "", mimeType, deflated, true);
+      content.data(), content.size(), "", mimeType, deflated, false);
```

Comment:
Why this ?

The content of the response is entirely defined by the request (the name of the zim and the `term` to search for), so we can cache it.

The only way the content change with the same url is that someone change the zim but keep the same name as before. And if we want to handle this case, we should not cache any responses.

Reply:
makes sense, let me fix this, thank you